src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -20,26 +23,39 @@
 public class SampleDataUtil {
     public static final Remark EMPTY_REMARK = new Remark("");
 
+    @SuppressWarnings("checkstyle:Indentation")
     public static Person[] getSamplePersons() {
         return new Person[]{
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
                         new Address("Blk 30 Geylang Street 29, #06-40"), EMPTY_REMARK,
-                        getTagSet("friends")),
+                        getTagSet("friends"),
+                        new DateOfCreation(LocalDate.of(2024, 1, 1)),
+                        new History(LocalDate.of(2024, 1, 1))),
```

Comment:
Looks like this file has the most amount of missing code coverage, try to add for ~half of them then can alr

Reply:
Because this is a util class and has no corresponding test class